student-bios/Verousca_bio.md

```diff
@@ -0,0 +1,18 @@
+# Hey there! I'm Verousca
+
+I am a Cameroonian living in Belgium. I am an IT student in the beginners
+stage. I have high interest in web development and computer engineering.
+
+## Hobbies
+
+- Travelling
+- Listening to Music
+- Escape room
+  
+## Photo
+
+![This is a alt text.](images/Verousca_Img.jpeg "This is a sample image.")
```

Comment:
It should be` /student-bios/images/Verousca_Img.jpeg`

Reply:
Thanks for the feedback. I have made the corrections.